code/game/objects/structures/containers.dm

```diff
@@ -14,44 +14,47 @@
 
 	AddComponent(/datum/component/seethrough, SEE_THROUGH_MAP_SHIPPING_CONTAINER)
 
-/obj/structure/shipping_container/nanotrasen
-	name = "\improper Nanotrasen shipping container"
-	desc = "A standard-measure shipping container for bulk transport of goods. This one prominently features Nanotrasen's logo, and so presumably could be carrying anything."
-	icon_state = "nanotrasen"
+/obj/structure/shipping_container/conarex
+	name = "\improper Conarex Aeronautics shipping container"
```

Comment:
Who is Conarex Aeronautics? I thought it was a syndicate faction but I don't see them mentioned in https://github.com/tgstation/common_core/blob/master/Governments%20and%20Organisations/Corporations/The%20Syndicate/The%20Syndicate.md

Not anything serious I just wanna know

Reply:
lol man idk - that was the default in there so i just left it - maybe some distant lore? If you can think of any i wouldnt mind adding or editing